docker/test_base/Dockerfile

```diff
@@ -8,6 +8,7 @@ ENV OTA_CLIENT_DIR="/ota-client"
 ENV OTA_IMAGE_SERVER_ROOT="/ota-image"
 ENV OTA_IMAGE_DIR="${OTA_IMAGE_SERVER_ROOT}/data"
 ENV VENV="${OTA_CLIENT_DIR}/.venv"
+ENV SPECIAL_FILE="path;adf.ae?qu.er\y=str#fragement"
```

Comment:
@Bodong-Yang 
Can you add multi-byte character file name?

Reply:
@obi-t4 Updated in [014e83b](https://github.com/tier4/ota-client/pull/170/commits/014e83bdb1616b00f2c5af288482a011d1cd696a).